geocoder/src/main/scala/grasshopper/geocoder/api/GeocodeFlow.scala

```diff
@@ -28,16 +28,20 @@ trait GeocodeFlow extends AddressPointsGeocode with CensusGeocode with Paralleli
       }
   }
 
-  def parsedInputAddressFlow: Flow[ParsedAddress, ParsedInputAddress, Unit] = {
+  def parsedInputAddressFlow: Flow[ParsedAddress, SearchableAddress, Unit] = {
     Flow[ParsedAddress]
-      .map(a =>
-        ParsedInputAddress(
-          a.parts.addressNumber,
-          a.parts.streetName,
-          a.parts.city,
-          a.parts.zip,
-          a.parts.state
-        ))
+      .map(parsed => {
```

Comment:
This notation is a bit weird. Instead of a `(` and a `{` after the `parsed =>`, the whole lambda block can be surrounded by `{`. Example: 

```Scala
.map { parsed => 

        val partMap: Map[String, String] = parsed.parts.map(part => (part.`type`, part.value)).toMap

        SearchableAddress(
          partMap.getOrElse("address_number_full", ""),
          partMap.getOrElse("street_name_full", ""),
          partMap.getOrElse("city_name", ""),
          partMap.getOrElse("zip_code", ""),
          partMap.getOrElse("state_name", "")
        )
     }
```